### PR TITLE
Add information on how to debug with IntelliJ and SDK 6.

### PR DESCRIPTION
**Pull request by bguedas**

### CODE REVIEW

File: SDK6UserGuide/runOnSimulator.rst [hunk 1]
```diff
@@ -75,19 +72,33 @@ The port can be changed by using the System Property ``debug.port``::
...
-- Add a breakpoint in your Application code.
-- Click on ``Run > Debug Configurations...``.
-- Select ``Remote Java Application``.
-- Click on the ``New launch configuration`` button.
-- Give a name to the launcher in the ``Name`` field.
-- Set the debug host and port.
-- Click on the ``Debug`` button.
+.. tabs::
 
+   .. tab:: IntelliJ IDEA
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/runOnSimulator.rst:82]: We must add Android Studio now.

**Review by tdelhomenie-microej — CHANGES_REQUESTED** [review]

File: SDK6UserGuide/runOnSimulator.rst (hunk 1 above)
Comment (bguedas) [reply SDK6UserGuide/runOnSimulator.rst]: I add "Android Studio /" in the tab's name. i don't have Android Studio installed to check if it's the same, but I guess it is.

Comment (gjadi-mej) [reply SDK6UserGuide/runOnSimulator.rst]: Yup, it's the same.

**Comment by tdelhomenie-microej** [comment]
@bguedas Archi 8.1 minimum for Android Studio / IntelliJ

**Comment by bguedas** [comment]
> @bguedas Archi 8.1 minimum for Android Studio / IntelliJ

Done.

**Review by tdelhomenie-microej — APPROVED** [review]

**Review by jcourcelmicroej — APPROVED** [review]

**Pull request merged**